### PR TITLE
Se 465 create a markdown file for the implementation of the minimum transaction rule

### PR DESCRIPTION
**Pull request by TJ-Everett**
Also corrected in-line links in all the docs

### CODE REVIEW

File: docs/userGuides/rules/MINIMUM-TRANSACTION.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
+# Minimum Transaction Rule
+
+## Purpose
+
+The purpose of the minimum-transaction rule is to prevent micro-trades or dust trades within an ecosystem. A developer can set the minimum number of tokens required per transfer to prevent these types of trades from occuring.
```
Comment (oscarsernarosero) [inline docs/userGuides/rules/MINIMUM-TRANSACTION.md]: Just a suggestion, I think you can get rid of the "from occurring" without affecting the meaning.

**Review by oscarsernarosero — APPROVED** [review]
Looks great Tayler.

**Review by mpetersoCode55 — APPROVED** [review]

File: docs/userGuides/rules/MINIMUM-TRANSACTION.md (hunk 1 above)
Comment (ShaneDuncan602) [reply docs/userGuides/rules/MINIMUM-TRANSACTION.md]: I agree

File: test/protocol/diamond/util/DiamondTestUtil.sol [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        RuleProcessorDiamondArgs memory diamondArgs = RuleProcessorDiamondArgs({
+            init: address(diamondInit),
+            // NOTE: "interfaceId" can be used since "init" is the only function in IDiamondInit.
+            initCalldata: abi.encode(type(IDiamondInit).interfaceId)
+        });
+
+        // Deploy the diamond.
+        return new RuleProcessorDiamond(_facetCuts, diamondArgs);
+    }
+}
```
Comment (ShaneDuncan602) [inline test/protocol/diamond/util/DiamondTestUtil.sol]: Didn't we delete this previously?

Comment (TJ-Everett) [reply test/protocol/diamond/util/DiamondTestUtil.sol]: Yes, Great catch. I get that wiped out.

**Review by ShaneDuncan602 — APPROVED** [review]
Looks good.

**Pull request merged**